locales/en-US/app.ftl

```diff
@@ -30,9 +30,19 @@ Home--load-from-url-submit-button =
     .value = Load
 
 Home--documentation-button = Documentation
+Home--menu-button = Enable Profiler Menu Button
```

Comment:
Can we create a new term, and use it here instead of Profiler?

Add at the beginning of the file

```
-profiler-brand-short-name = Profiler
```

Here

```
Home--menu-button = Enable { -profiler-brand-short-name } Menu Button

```